dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/ws/crypto/util/PasswordCipherUtil.java

```diff
@@ -216,7 +218,7 @@ public static byte[] decipher(byte[] encrypted_bytes,
 
         byte[] decrypted_bytes = null;
 
-        if (AES.equalsIgnoreCase(crypto_algorithm)) {
+        if (AES.equalsIgnoreCase(crypto_algorithm) || AES_128.equalsIgnoreCase(crypto_algorithm) || AES_256.equalsIgnoreCase(crypto_algorithm)) {
```

Comment:
i think the check for `AES_128` and `AES_256` can be removed here because you are normalizing the crypto tag to be `aes` in `PasswordUtil.java` on the creation side, so it can never be `aes-128` nor `aes-256` when we receive it.

Reply:
I am doing this intentionally for future support